deployment/scripts/run_dcl_node

```diff
@@ -0,0 +1,459 @@
+        usage
+        exit 1
+    fi
+
+    NODE_NAME="$1"
+
+    if [[ -z "$KEY_NAME" ]]; then
+        KEY_NAME="${NODE_NAME}admin"
+    fi
+
+    if [[ "$NODE_TYPE" == "genesis" ]]; then
+        PEER_ADDR="localhost"
+    else
+        if [[ ! -f "$PERSISTENT_PEERS_FILE" ]]; then
+            echo "Error. '$PERSISTENT_PEERS_FILE' file not found"
+            exit 1
+        fi
+
+        if [[ ! -f "$GENESIS_IN_FILE" ]]; then
+            echo "Error. '$GENESIS_IN_FILE' file not found"
+            exit 1
+        fi
+
+        PERSISTENT_PEERS="$(cat $PERSISTENT_PEERS_FILE | head -n1)"
+
+        if [[ -z "$PEER" ]]; then
+            PEER="$(echo "$PERSISTENT_PEERS" | awk -F',' '{print $1}')"
+        fi
+        PEER_ID="$(echo "$PEER" | awk -F'@' '{print $1}')"
+        PEER_HOSTSPEC="$(echo "$PEER" | awk -F'@' '{print $2}')"
+        PEER_ADDR="$(echo "$PEER_HOSTSPEC" | awk -F':' '{print $1}')"
+    fi
+}
+
+function config_cli {
+    set -eu
+    set -o pipefail
+
+    local _chain_id="$1"
+    local _peer_host="${2:-}"
+
+    dclcli config chain-id "$_chain_id"
+    dclcli config output json
+    dclcli config indent true
+    dclcli config trust-node false
+    if [[ -n "$_peer_host" ]]; then
+        dclcli config node tcp://"$_peer_host"
+    fi
+}
+
+
+function add_key {
+    set -eu
+    set -o pipefail
+
+    local _key_name="$1"
+    local _output="${2:-${_key_name}.dclkey.json}"
+
+    dclcli keys add "$_key_name" 2>&1 | tee "$_output"
+    echo -e "\tSaved key data in '$_output'"
+}
+
+
+function node_init {
+    set -eu
+    set -o pipefail
+
+    local _name="$1"
+    local _chain_id="$2"
+
+    rm -rf "$HOME/.dcld/config/gentx"
+    rm -f "$GENESIS_FILE"
+
+    dcld init "$_name" --chain-id "$_chain_id"
+}
+
+
+function build_genesis {
+    set -eu
+    set -o pipefail
+
+    local _key_name="$1"
+    local _acc_address="$2"
+    local _acc_pubkey="$3"
+    local _acc_roles="${4:-Trustee,NodeAdmin}"
+
+    dcld add-genesis-account --address="$_acc_address" --pubkey="$_acc_pubkey" --roles="$_acc_roles"
+    dcld gentx --from "$_key_name"
+    dcld collect-gentxs
+    dcld validate-genesis
+}
+
+function config_node {
+    set -eu
+    set -o pipefail
+
+    local _node_user="$1"
+    local _dcld_service_p="${2:-./dcld.service}"
+    local _peers="${3:-}"
+
+    local _debug="${DEBUG:-}"
+
+    sed -i -r 's~^laddr = "tcp://127.0.0.1:~laddr = "tcp://0.0.0.0:~' "$CONFIG_FILE"
+
+    # FIXME verify
+    if [[ -n "$_peers" ]]; then
+        sed -i -r "s~^persistent_peers = .*~persistent_peers = \"$_peers\"~" "$CONFIG_FILE"
+    fi
+
+    if [[ -z "$_debug" ]]; then
+        # config the service
+        # Ubuntu only
+        grep -q -i -e 'ubuntu' /etc/*-release && {
+            if [[ ! -f "$_dcld_service_p" ]]; then
+                echo "Error. '$_dcld_service_p' file not found"
+                exit 1
+            fi
+
+            # Open '26656' (p2p) and '26657' (RPC) ports. 
+            sudo ufw allow 26656/tcp
```

Comment:
Makes sense to move to a separate script

Reply:
added TODO there